### PR TITLE
Kava

### PR DESCRIPTION
**Pull request by HustinKava**
Added Jquery to index.html
Created basic javascript code to be able to pull data from the new york times movie reviews api.

### CODE REVIEW

**Review by hector-gm — APPROVED** [review]
Thanks for the update

File: assets/javascript/script.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var tmdbApi = 'a4e5136717cef17c7b2d9c9331196e91';
+var nytApi = 'dWR7musz8J5oEwVOdiYZeCsFGdcOnDDO';
+
+var askURL = 'https://api.nytimes.com/svc/movies/v2/reviews/search.json?query=godfather&api-key=' + nytApi;
```
Comment (chensun113) [inline assets/javascript/script.js:4]: This is a API

Comment (chensun113) [reply assets/javascript/script.js:4]: a URL

**Review by chensun113 — COMMENTED** [review]
good start

**Comment by chensun113** [comment]
A good start (rica is testing)

**Pull request merged**